consts/user.js

```diff
@@ -0,0 +1,19 @@
+const allowedUserFieldsForUpdate = {
+  photo: true,
+  firstName: true,
+  lastName: true,
+  address: {
+    country: true,
+    city: true
+  },
+  city: true,
+  experience: true,
+  subject: true,
+  averageRating: true,
```

Comment:
this field will not exist, i removed it in my PR and even if it stayed, this field wouldn't have belonged to this filter.